packages/ra-ui-materialui/src/list/BulkActionsToolbar.tsx

```diff
@@ -48,7 +48,7 @@ export const BulkActionsToolbar = (props: BulkActionsToolbarProps) => {
                         className={BulkActionsToolbarClasses.icon}
                         aria-label={translate('ra.action.unselect')}
                         title={translate('ra.action.unselect')}
-                        onClick={onUnselectItems}
+                        onClick={() => onUnselectItems()}
```

Comment:
why do you need this? This will force a rerender of the button.

Reply:
Move inside a useCallback